playbooks/files/plugins/container_storage_check.py

```diff
@@ -87,7 +87,6 @@ def main():
         status_err(str(e))
     else:
         status_ok()
-    finally:
```

Comment:
It looks like this raises an exception https://github.com/rcbops/rpc-maas/blob/master/playbooks/files/plugins/maas_common.py#L563-L571 which would produce a stacktrace making it impossible for maas to record the metrics even if something was produced.